src/v2/server.ts

```diff
@@ -43,6 +43,27 @@ app.get(
   }
 )
 
+// Experiment with overwriting OneTrust consent cookie with server-side version
+// to get around Safari's 7 day limit for client-side cookies.
+app.get("/set-tracking-preferences", (req, res) => {
+  const { OptanonAlertBoxClosed, OptanonConsent } = req.query
+
+  const cookieConfig = {
+    maxAge: 1000 * 60 * 60 * 24 * 365,
```

Comment:
Github copilot suggested this /shrug. 1000 years?

Reply:
I kindof hope that there is some machine running somewhere in 1000 years that overflows this buffer.